examples/model_selection/plot_nested_cross_validation_iris.py

```diff
@@ -0,0 +1,119 @@
+"""
+=========================================
+Nested versus Non-Nested cross-validation
+=========================================
+
+This example compares non-nested and nested cross-validation strategies on a
+classifier of the iris data set. Nested cross-validation (CV) is often used to
+train a model in which hyperparameters also need to be optimized. Nested CV
+approximates the generalization error of the resulting estimator of the
+hyperparameter search. It is generally best practice to use nested CV as
+non-nested CV will sometimes provide a slightly more biased and optimistic
```

Comment:
I don't think we need "slightly" and "sometimes".

Choosing the parameters that maximise non-nested CV biases it to the dataset, yielding an overly-optimistic score.

This notion is repeated multiple times here and I think you need to work on making it succinct.